spec/factories/omniauth_providers.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :omniauth_provider, class: OmniAuth::AuthHash.name do
     provider 'default'
-    uid SecureRandom.random_number(2**48)
+    sequence(:uid) { |n| SecureRandom.random_number(2**48) + n }
```

Comment:
@lowjoel It should work now, previously it's because this UID is not updated during testing.